migration-guide/business-central.adoc

```diff
@@ -0,0 +1,129 @@
+include::../partials/attributes.adoc[]
+= Migrating from Business Central to IBM {DEV_TOOLS}
+
+== Overview
+
+Business Central was a web-based platform for authoring business automation assets such as data objects, rules, processes, cases, and planning entities. It included capabilities for monitoring, administration, and management of business automation assets.  With the release of IBM {DEV_TOOLS} as well as IBM {CANVAS}, which unlike Business Central was designed from the ground up to be cloud-native, most of these asset management capabilities have been replaced or in some cases take a different approach altogether.  This section covers migration of business assets created in Business Central so that they are able to be maintained by developers using either {DEV_TOOLS} and/or {CANVAS}.
+
+== Prerequisites
+
+The following tools are required in order to migrate from Business Central to {DEV_TOOLS}.  It is assumed that you have a current version of Business Central running in order to be able to export certain types of assets to their supported form in {PRODUCT_SHORT}:
```

Comment:
The following tools are required in order to migrate from Business Central to {DEV_TOOLS}.  It is assumed that you have a current version of Business Central running in order to be able to export certain types of assets to their supported form in {PRODUCT_SHORT}:
Proposed change: The following tools are required to migrate from Business Central to {DEV_TOOLS}. It is assumed that you have a current version of Business Central running to export certain types of assets to their supported form in {PRODUCT_SHORT}: